### ISSUE TITLE
Broken table generation

### ISSUE THREAD

**Issue by Faboor** [MikeCoder/markdown-preview.vim#13]
## Expected Behavior 
[TEST.md](../blob/master/TEST.md) example table should generate a `<table>`

## Actual Behavior
It generates a paragraph:
``` HTML
<h3>Table</h3>
<p>| Tables | Are | Cool | | ------------- |:-------------:| -----:| | col 3 is | right-aligned | $1600 | | col 2 is | centered | $12 | | zebra stripes | are neat | $1 |</p>
```

## Steps to Reproduce the Problem
  1. Install markdown-preview
  2. `vim ~/.vim/bundle/markdown-preview.vim/TEST.md`
  3. `:MarkdownPreview github`

## Specifications
Version: [40cf42a - update css file](../commit/40cf42ae24635303b2d41c1091b407fe8cbce39f)

**Comment by D0n9X1n** [comment]
Thanks for the bug reporting. This has been fixed in v3.1.3.

Use `:PlugUpdate` or other commands to upgrade the plugin.

**Comment by Faboor** [comment]
Can confirm, works now. Thanks!
